honeybadgermpc/ntl/helpers.pyx

```diff
@@ -158,16 +174,25 @@ cpdef vandermonde_batch_interpolate(x, data_list, modulus):
     for i in range(n_chunks):
         l = len(data_list[i])
         for j in range(l):
-            m[j][i] = py_obj_to_ZZ_p(data_list[i][j])
+            m[j][i] = intToZZp(data_list[i][j])
         for j in range(l, k):
-            m[j][i] = py_obj_to_ZZ_p(0)
+            m[j][i] = intToZZp(0)
     cdef mat_ZZ_p reconstructions
+    end_time = time.time()
```

Comment:
Are these still needed?

Reply:
Ah. Missed a few. They are not needed anymore. Removing them now.